### ISSUE TITLE
♿ AVT3 ♿ Date picker - User can't select date

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1588]
User tested with latest version of JAWS 2018, Windows 7, Chrome latest version

User cannot select date and must entered date manually. 

_"It is a problem when the user wants to know which day of the week or month is the particular date"_

**Comment by snidersd** [comment]
The date picker should also have a fully functioning validation alert so it can be a fully tested with a screen reader. [See WCAG Guidelines 3.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error.html)

**Comment by dakahn** [comment]
keyboard affordances are there _but they are undocumented_ so discoverability is clumsy. 👍
